src/Models/OrderModel.php

```diff
@@ -92,4 +92,64 @@ public function addOrder(OrderEntityInterface $orderEntity): bool
 
         return true;
     }
+
+    /**
+     * returns an array of all the orders in the DB with the products ordered as well or false if it fails.
+     * @return array|false
```

Comment:
need to update docs to include parameter
```suggestion
     * returns an array of incomplete or complete orders in the DB with the products ordered as well or false if it fails.
     * @param string $completed
     * @return array|false
```
```